package.json

```diff
@@ -237,6 +237,7 @@
   "scripts": {
     "test": "rimraf ./build && tsc && mocha --file ./build/compiled/test/utils/test-setup.js --bail --recursive --timeout 60000 ./build/compiled/test",
     "test-fast": "mocha --file ./build/compiled/test/utils/test-setup.js --bail --recursive --timeout 60000 ./build/compiled/test",
+    "test-github-issue": "mocha --file ./build/compiled/test/utils/test-setup.js --bail --recursive --timeout 60000 ./build/compiled/test/github-issues/9323",
     "compile": "rimraf ./build && tsc",
```

Comment:
Shouldn't be merged - you can use `--grep` option of `mocha` for you development.

Reply:
Awesome - you are right! I will update this today. Thanks for the reply!

Reply:
This has been updated @Ginden